pom.xml

```diff
@@ -746,6 +747,12 @@
       <version>${junit5.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-junit-jupiter</artifactId>
+      <version>${mockito4.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Move this dep to DepMgnt and add it as dep in component-tools instead.

Reply:
Just did it as junit5 was added.
I can add junit5 into depMgnt too.

Reply:
changed